tabular/src/autogluon/tabular/trainer/model_presets/presets.py

```diff
@@ -154,7 +154,18 @@ def get_preset_models(path, problem_type, eval_metric, hyperparameters, feature_
                 logger.log(20, f"\tFound '{model_type}' model in hyperparameters, but '{model_type}' is present in `excluded_model_types` and will be removed.")
                 continue  # Don't include excluded models
             if isinstance(model_cfg, str):
-                model_cfgs_to_process += get_preset_custom(name=model_cfg, problem_type=problem_type, num_classes=num_classes)
+                if model_type == 'AG_TEXT_NN':
+                    AG_TEXT_IMPORT_ERROR = 'autogluon.text has not been installed. ' \
+                                           'You may try to install "autogluon.text" ' \
+                                           'first by running. ' \
+                                           '`python3 -m pip install autogluon.text`'
+                    try:
+                        from autogluon.text import ag_text_presets
```

Comment:
Better to add a new try_except function where all the other try_except functions are, will look cleaner.